src/System.Net.Http/tests/FunctionalTests/IdnaProtocolTests.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Net.Test.Common;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Http.Functional.Tests
+{
+    public abstract class IdnaProtocolTests : HttpClientTestBase
+    {
+        private const string s_IdnaHost = "âbçdë.com";
```

Comment:
We scrubbed our source code during opensource to escape unicode chars in our files.. We should maintain consistency..

Reply:
Should we have more unicode test cases?

Reply:
Probably.  I'll add a few more.

Reply:
Maybe replace with `\u` prefixes.

Reply:
The updated commit uses \u.